packages/react/src/ProgressBar/ProgressBar.tsx

```diff
@@ -60,37 +57,76 @@ const ProgressContainer = styled.span<StyledProgressContainerProps>`
   ${sx};
 `
 
+export type ProgressBarItems = React.HTMLAttributes<HTMLSpanElement> & {'aria-label'?: string} & ProgressProp & SxProp
+
+export const Item = forwardRef<HTMLSpanElement, ProgressBarItems>(
+  (
+    {progress, 'aria-label': ariaLabel, 'aria-valuenow': ariaValueNow, 'aria-valuetext': ariaValueText, ...rest},
+    forwardRef,
+  ) => {
+    const progressAsNumber = typeof progress === 'string' ? parseInt(progress, 10) : progress
+
+    const ariaAttributes = {
+      'aria-valuenow':
+        ariaValueNow || (progressAsNumber !== undefined && progressAsNumber >= 0 ? Math.round(progressAsNumber) : 0),
```

Comment:
Small suggestion for `??` over `||` for fallback values, no real different though 👍 

```suggestion
        ariaValueNow ?? (progressAsNumber !== undefined && progressAsNumber >= 0 ? Math.round(progressAsNumber) : 0),
```

Reply:
`??` looks cooler, so let's roll with it 😎